web/autograder_service.go

```diff
@@ -21,18 +22,20 @@ type AutograderService struct {
 	logger *zap.SugaredLogger
 	db     database.Database
 	scms   *auth.Scms
-	bh     BaseHookOptions
+	// bh     BaseHookOptions
```

Comment:
Better to remove these lines instead of commenting. Should be easy to see the diff.